src/main/webapp/WEB-INF/tags/page.tag

```diff
@@ -29,7 +29,7 @@
     <script type="text/javascript" src="${data.jQueryUiFilePath}"></script>
     <script type="text/javascript" src="/js/common.js"></script>
     <script type="text/javascript" src="/bootstrap/js/bootstrap.min.js"></script>
-
+    <script type="text/javascript" src="http://github.com/bartaz/sandbox.js/raw/master/jquery.highlight.js"></script>
```

Comment:
I think we shouldn't use files from other sites. Instead, if possible, storing it in /js/ folder could be better.